server/broadcast.go

```diff
@@ -38,6 +41,9 @@ var Policy *verification.Policy
 var BroadcastCfg = &BroadcastConfig{}
 var MaxAttempts = 3
 
+var MetadataQueue event.Producer
+var MetadatPublishTimeout = 1 * time.Second
```

Comment:
```suggestion
var MetadataPublishTimeout = 1 * time.Second
```
Will also need to change wherever it's being used.